p11-kit/list-objects.c

```diff
@@ -43,8 +43,10 @@
 #include "message.h"
 #include "print.h"
 #include "tool.h"
+#include "uri.h"
 
 #include <assert.h>
+#include <stdbool.h>
```

Comment:
Isn't this already included through "compat.h", which also provide fallback definitions of true / false?